calculators/hand.py

```diff
@@ -44,3 +51,28 @@ def process(self):
                 self.set_output(0, ImageData(nf, image.timestamp))
                 return True
         return False
+
+
+class HandThumbDetector(Calculator):
+
+    def __init__(self, name, s, options=None):
+        super().__init__(name, s, options=options)
+
+    def process(self):
+        hand_data = self.get(0)
+        if hand_data:
+            kp, box = hand_data
+            palm = kp[0][1]
+            thumb = kp[4][1]
+            fingers = [kp[8][1], kp[12][1], kp[16][1], kp[20][1]]
```

Comment:
We should define some nice way of referring to this rather than using the numbers here. Not needed now but for a future PR.

Reply:
Yes, I would really like that. For now I just used what the HandDetector node output.